src/styles/getTooltipStyles.js

```diff
@@ -122,6 +122,13 @@ export default function getTooltipStyles(
     // from displaying too near to the edge of the screen.
     const horizontalShift = computeHorizontalShift(windowWidth, xOffset, componentWidth, tooltipWidth, manualShiftHorizontal);
 
+    // Determine if we need to shift the pointer horizontally to prevent it
+    // from displaying too near to the edge the tooltip by
+    // shift it to right a bit if the tooltip is positioned on the extreme left
+    // or shift it to left a bit if the tooltip is positioned on the extreme right.
```

Comment:
```suggestion
    // Determine if we need to shift the pointer horizontally to prevent it from being too near to the edge of the tooltip
    // We shift it to the right a bit if the tooltip is positioned on the extreme left
    // and shift it to left a bit if the tooltip is positioned on the extreme right.
```

Reply:
The first line can be longer since the next line of code is long anyway